patches/base-path.diff

```diff
@@ -255,15 +255,7 @@ Index: code-server/lib/vscode/src/vs/code/browser/workbench/workbench.ts
 ===================================================================
 --- code-server.orig/lib/vscode/src/vs/code/browser/workbench/workbench.ts
 +++ code-server/lib/vscode/src/vs/code/browser/workbench/workbench.ts
-@@ -484,6 +484,7 @@ function doCreateUri(path: string, query
- 		});
- 	}
- 
-+	path = (window.location.pathname + "/" + path).replace(/\/\/+/g, "/")
```

Comment:
I am not sure the authentication callbacks are still used in code-server, but we might want to add this back.  Looks like it is found on line 147 now.

Reply:
The following commits had a significant impact on patches:

* https://github.com/microsoft/vscode/commit/322bc2d7d86e8d0c2ccb1bfdabe594e2011d500f
  * File: `lib/vscode/src/vs/code/browser/workbench/workbench.ts`
  * Patches: `base-path.diff`, `proxy-uri.diff`
* https://github.com/microsoft/vscode/commit/ef42b49e806b4a4f1202ec2e31f441465a7047de
  * File: `lib/vscode/src/vs/server/node/serverServices.ts`
  * Patch: `telemetry.diff`

I am not sure I updated the patches correctly.